scripts/compose-dev.yml

```diff
@@ -21,6 +21,8 @@ services:
 
   postgres:
     image: centos/postgresql-95-centos7 
+    ports:
+    - '2345:5432'
```

Comment:
Expose the postgres port so that local client tools can be used, and the restore DB playbook can run locally. Using host port 2345 so that it hopefully won't collide with anything on Travis.